catalog.html

```diff
@@ -0,0 +1,197 @@
+                  "Мотель"
+                  </label>
+                  <input type="checkbox" name="apartments" id="apartments" checked>
+                  <label for="apartments">
+                  "Апартаменты"
+                  </label>
+               </div>
+               <div>
+                  <p>Стоимость в сутки (Р):</p>
+                  <div>
+                     <p>
+                        "от"
+                        <span id="min-price">0</span>
+                     </p>
+                     <p>
+                        "до"
+                        <span id="max-price">3000</span>
+                     </p>
+                  </div>
+                  <div>
+                     <div id="range-slider">
+                        <div data-handle="0" tabindex="0" role="slider">
+                        </div>
+                        <div data-handle="1" tabindex="0" role="slider">
+                        </div>
+                     </div>
+                  </div>
+                  <button type="submit">Показать</button>
+               </div>
+            </form>
+         </section>
+         <section>
+            <h2>Доступные отели:</h2>
+            <div>
+               <p>
+                  "Найдено: "
+                  <span id="hotelNumber">3</span>
+               </p>
+               <p>Сортировка:</p>
+               <ul>
+                  <li>
+                     <a>По цене</a>
+                  </li>
+                  <li>
+                     <a href="#">По типу</a>
+                  </li>
+                  <li>
+                     <a href="#">По рейтингу</a>
+                  </li>
+               </ul>
+               <button type="button" title="Сортировать по убыванию"></button>
+               <button type="button" title="Сортировать по возрастанию"></button>
+            </div>
+            <ul>
+               <li>
+                  <div>
+                     <img src="https://via.placeholder.com/135x90" alt="Amara spa">
+                  </div>
+                  <div>
+                     <a href="#">
+                        <h2>Amara Resort & Spa</h2>
+                     </a>
+                     <div>
+                        <p>Гостиница</p>
+                        <p>от 4000 Р.</p>
+                     </div>
+                     <a href="#">Подробнее</a>
+                     <a href="#">Забронировать</a>
+                  </div>
+                  <div>
+                    <div title="4 звезды">
+                        <img src="https://via.placeholder.com/18x17" alt="★">
+                        <img src="https://via.placeholder.com/18x17" alt="★">
+                        <img src="https://via.placeholder.com/18x17" alt="★">
+                        <img src="https://via.placeholder.com/18x17" alt="★">
+                    </div>
+                     <p>Рейтинг: <span>8,5</span></p>
+                  </div>
+               </li>
```

Comment:
Изображение само по себе блочный элемент, ему обертка не нужна. Также заменим звезды, как я [писал в прошлом пр](https://github.com/htmlacademy-htmlcss/1706635-sedona-30/pull/2#discussion_r563851074)

```suggestion
               <li>
                  <img src="https://via.placeholder.com/135x90" alt="Amara spa">
                  <div>
                     <a href="#">
                        <h2>Amara Resort & Spa</h2>
                     </a>
                     <div>
                        <p>Гостиница</p>
                        <p>от 4000 Р.</p>
                     </div>
                     <a href="#">Подробнее</a>
                     <a href="#">Забронировать</a>
                  </div>
                  <div>
                    <span title="4 звезды">★★★★</span>
                     <p>Рейтинг: <span>8,5</span></p>
                  </div>
               </li>
```